src/pages/gutenberg/free-audio-books.vue

```diff
@@ -364,6 +350,12 @@ export default {
     handleInputChange(value) {
       logTrackerEvent(this, 'Gutenberg', 'inputChange', value.target.value, 1);
     },
+    debouncedUpdateDisplayNumber: debounce(
```

Comment:
Is it better to debounce "setting input to `searchKeyword`" instead of this number? We dont need per character update on the result

Reply:
[🎨 Use debounce for setting searchKeyword](https://github.com/likecoin/liker-land/pull/1639/commits/5569cf0c6adc94c6eba29236e53631073b015fd0)